egui-winit/src/lib.rs

```diff
@@ -458,6 +458,9 @@ impl State {
             // https://github.com/rust-windowing/winit/issues/1695 being closed
             delta.x *= -1.0;
         }
+        if cfg!(target_os = "windows") {
+            delta.x *= -1.0;
```

Comment:
if this is wrong on mac and windows, perhaps it is also wrong on linux? i.e perhaps we should just always invert `delta.x`no matter what platform

Reply:
I agree. However, it's not clear to me how we find the correct answer. As mentioned in #356, it appears that it's not even consistent across Linux desktop environments.

The only dev environment I have readily available is windows. I didn't want to cause problems on systems I can't test, and I wanted to keep this PR tightly scoped.

Most importantly, this is really a workaround for an upstream winit issue. If I can get this https://github.com/rust-windowing/winit/pull/2101 merged, the egui issue and PR for horizontal scrolling on Windows can be closed.